native_client/swift/deepspeech-ios.podspec

```diff
@@ -0,0 +1,18 @@
+# Pull in version from outside
+version = File.read(File.join(__dir__, "../../training/deepspeech_training/VERSION")).split("\n")[0]
+
+Pod::Spec.new do |s|
+  s.name         = "deepspeech-ios"
+  s.version      = version
+  s.summary      = "DeepSpeech"
+  s.homepage     = "https://github.com/mozilla/DeepSpeech"
+  s.license      = "Mozilla Public License 2.0"
+  s.authors      = "Mozilla et al."
+
+  s.platforms    = { :ios => "13.5" }
+
+  # TODO: Consider what we want to do here
+  s.source       = { :git => "", :tag => "#{s.version}" }
```

Comment:
The proper tag would be `v#{s.version}` but maybe you want to refer the commit directly ?

Reply:
Yeah I honestly don't know how this works with publishing to cocoa pods. I've only used it for internal pods in which case this is omitted. Does it assume the lib is in the root dir? This was meant to be a starting point, but I think the podspec has to be modified to publish it still.

Reply:
I have no idea at all about what the podspec is, what a cocoa pod is :). But if `s.source` is meant to designate the exact source code reference, then it should be the commit directly.

Reply:
Yes, that makes sense. I've also made the paths relative assuming a clone from the repo root

Reply:
And I _think_ that resolves this TODO. Again, my goal was just to provide a starting point for whoever actually submits to the specs repo, but I think this should be good now.